src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.4.3
++++++
```

Comment:
These 2 lines are not necessary, otherwise you need to touch the version inside of setup.py as well. Just remove them.

Reply:
Fixed 👍